ahcore/writers.py

```diff
@@ -170,6 +175,8 @@ def init_writer(self, first_coordinates: GenericArray, first_batch: GenericArray
             "format": _format,
             "dtype": str(first_batch.dtype),
             "is_binary": self._is_compressed_image,
+            "precision": self._precision.value if self._precision else "FP32",
```

Comment:
Is this "FP32" correct? In the type class you have "float32", why not use your enum here? I.e. make it something like `str(InferencePresicion.FP32)`?